teach/templates/clubs/email/create_club_staff/body.txt

```diff
@@ -0,0 +1,14 @@
+Club captain {{ full_name }} would like to add their Club to the map (owned by {{ username }}).
```

Comment:
@gideonthomas - I don't have a preference for what the email says since I have to log in and look at their application anyway. This seems fine - thanks!